### PR TITLE
Fixed shadowing of builtin function type()

### PR DESCRIPTION
**Pull request by Eloque**
Removed unused 'type = 'comic' from import function
Renamed type to search_type in searchit()
Renamed type to status_type in clearhistory()

Having variables with the same name as the builtin function caused errors on import. Made suggested fix by renaming variables.

### CODE REVIEW

**Comment by evilhero** [comment]
Just a headsup - that changing those particular variable names will break the following aspects:
- clearing logs via the history tab (changing the type variable to new one in the ```history.html``` - all the references are in the subhead_menu aspect and make specific reference to the ```type``` variable name)
- storyarc searches via the search bar (need to update the ```type``` in ```<input type="hidden" name="type" value="story_arc">``` to point to the new variable name of ```search_type```)

While these don't break anything, we should probably keep the naming identical throughout - like for searchit, you're changing the variable but then sending it as something else - which works, but just for consistency as I'm not 100% sure if other calls utilise the searchit function:
- searchit as mentioned above in webserve.py - just remove all references to type and replace with ```search_type``` for consistency.
- the searchresults.html needs to update the variable name as well in relation to the the 1st bullet (most for the checks are like ```if type == 'story_arc'```, but there are several in that html file, so changing the searchit to a consisten ```search_type``` means that the html should reference it as well).


Other than those minor things, this is good once those are addressed.

Cheers (and thanks)!

**Comment by Eloque** [comment]
Updated the HTML
Update of webserve.py was done in original commit.

**Review by evilhero — CHANGES_REQUESTED** [review]

File: data/interfaces/default/history.html [hunk 1]
```diff
@@ -6,10 +6,10 @@
 <%def name="headerIncludes()">
 	<div id="subhead_container">
 		<div id="subhead_menu">
-			<a id="menu_link_delete" href="#" onclick="doAjaxCall('clearhistory?type=all',$(this),'table')" data-success="All History cleared">Clear All History</a>
-			<a id="menu_link_delete" href="#" onclick="doAjaxCall('clearhistory?type=Processed',$(this),'table')" data-success="All Processed cleared">Clear Processed</a>
-			<a id="menu_link_delete" href="#" onclick="doAjaxCall('clearhistory?type=Unprocessed',$(this),'table')" data-success="All Unprocessed cleared">Clear Unprocessed</a>
-			<a id="menu_link_delete" href="#" onclick="doAjaxCall('clearhistory?type=Snatched',$(this),'table')" data-success="All Snatched cleared">Clear Snatched</a>
+			<a id="menu_link_delete" href="#" onclick="doAjaxCall('clearhistory?search_type =all',$(this),'table')" data-success="All History cleared">Clear All History</a>
```
[the remark this answers is not in the document]
Comment (Eloque) [reply data/interfaces/default/history.html]: No idea how that got in there. Fixed now.

**Comment by barbequesauce** [comment]
Looks ok to me following last commit - @evilhero  ?

**Review by evilhero — APPROVED** [review]

**Pull request merged**